vllm_hpu_extension/rotary_embed.py

```diff
@@ -125,6 +138,63 @@ def forward(self, positions: torch.Tensor, query: torch.Tensor,
                  (key.shape[0], key.shape[1], key.shape[2] * key.shape[3]))
 
 
+class HpuLinearScalingRotaryEmbedding(HpuRotaryEmbedding):
+
```

Comment:
Add a short help string describing what is different here compared to LinearScalingRotaryEmbedding.